src/System.CommandLine/README.md

```diff
@@ -0,0 +1,227 @@
+# System.CommandLine
+
+System.CommandLine provides robust support for command-line parsing, invocation, and shell completions in .NET applications. It supports both POSIX and Windows conventions, making it easy to build professional command-line interfaces.
+
+## Getting Started
+
+### Basic Command
+
+Here's a simple "Hello World" command-line application:
+
+```csharp
+using System.CommandLine;
+
+var rootCommand = new RootCommand("Sample command-line app");
+
+var nameOption = new Option<string>(
+    aliases: ["--name", "-n"],
```

Comment:
This constructor call doesn't match the latest signature:

https://github.com/dotnet/command-line-api/blob/0c37e9e6823c3a71f7710f9ef660ed40618390c2/src/System.CommandLine/Option%7BT%7D.cs#L19-L22